packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -992,7 +992,8 @@ const AccountHeader = React.memo(
   },
 );
 
-function AllTransactions({ transactions, filtered, children }) {
+function AllTransactions({ account = {}, transactions, filtered, children }) {
```

Comment:
When does it make sense to leave the `account` prop off?

Reply:
When you're viewing `all accounts` or `all on/off budget accounts`.